lib/puppet/type/openldap_database.rb

```diff
@@ -101,4 +101,18 @@ def should_to_s( newvalue )
     newvalues(:true, :false)
     defaultto(:true)
   end
+
+  newproperty(:mirrormode) do
```

Comment:
You should use `:boolean => true, :parent => Puppet::Parameter::Boolean` here : https://docs.puppetlabs.com/guides/custom_types.html#boolean-parameters